test/tensor_util.hpp

```diff
@@ -74,114 +71,4 @@ void operate_over_subtensor_impl(const data_operator_t<T>& r_data_operator,
     }
 }
 
-template <typename T>
```

Comment:
Why did we delete these function?

Reply:
Looks like it was used for debugging, or someone accidentally left it here. In any case, this code is not used anywhere, and I decided to remove it.